packages/viewer/src/modules/UrlHelper.ts

```diff
@@ -0,0 +1,276 @@
+import { SpeckleViewer } from '@speckle/shared'
+
+interface ReferencedObjectUrl {
+  origin: string
+  projectId: string
+}
+
+interface CommitReferencedObjectUrl {
+  origin: string
+  streamId: string
+  commitId: string
+}
+
+export default class UrlHelper {
```

Comment:
This isn't a problem that necessarily needs fixing, but I don't see the point of this being a class, especially if all of its functions are static. These could just easily be functions directly exported out from the module.